relnotes/timedivision.breaking.md

```diff
@@ -0,0 +1,5 @@
+### Round relative time interval arguments
+
+`ocean.text.arguments.TimeIntervalArgs`
+
+isTimeInterval ( cstring value ) is now deprecated
```

Comment:
A method being deprecated itsn't a breaking change.

Reply:
Agreed. Should be `timedivision.deprecation.md`.

Reply:
done :)